index.html

```diff
@@ -0,0 +1,504 @@
+                </figure>
+
+                <figure>
+                    <img src="https://static01.nyt.com/images/2019/09/10/science/10INDIAMOON1/10INDIAMOON1-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="">
+                    <figcaption>Manjunath Kiran/Agence France-Presse — Getty Images. Did India’s Chandrayaan-2 Moon Lander Survive? The Chances Are Slim <br /> Sept. 10</figcaption>
+                </figure>
+                <!-- <h4>Editors' Picks</h4> -->
+                <figure>
+                    <img src="https://static01.nyt.com/images/2019/10/06/magazine/06-mag-maddow-cover/06-mag-maddow-cover-threeByTwoSmallAt2X-v2.jpg?quality=75&auto=webp&disable=upscale" alt="">
+                    <figcaption>Christopher Griffith for The New York Times. This Is the Moment Rachel Maddow Has Been Waiting For <br /> Oct. 1</figcaption>
+                </figure>
+
+                <figure>
+                    <img src="https://static01.nyt.com/images/2019/09/27/arts/27snl/27snl-threeByTwoSmallAt2X-v2.jpg?quality=75&auto=webp&disable=upscale" alt="">
+                    <figcaption>NBC, via Associated Press <br />‘S.N.L.’ Is Sorry: A Brief History of the Show’s (Sort Of) Apologies <br /> Sept. 27</figcaption>
+                </figure>
+
+                <figure>
+                    <img src="https://static01.nyt.com/images/2019/09/29/magazine/29-mag-Presidential-Busts-06/29-mag-Presidential-Busts-06-threeByTwoSmallAt2X-v3.jpg?quality=75&auto=webp&disable=upscale" alt="">
+                    <figcaption>Hannah Price for The New York Times. A Guided Tour Through the Ruins of America's Democracy <br /> Sept. 24</figcaption>
+                </figure>
+
+            </div>
+            <div class="right-more">
+                <ul>
+                    <li><a href="#">The Evolution of the Hard Hat</a></li>
+                    <li><a href="#">Woodbury Common: Bargain Hunting in the Fashion Afterlife</a></li>
+                    <li><a href="#">Bernie Sanders Is Hospitalized for Heart Procedure and Cancels Events</a></li>
+                    <li><a href="#">Former Dallas Police Officer Is Guilty of Murder for Killing Her Neighbor</a></li>
+                    <li><a href="#">Shoot Migrants’ Legs, Build Alligator Moat: Behind Trump’s Ideas for Border</a></li>
+                    <li><a href="#">Opinion: If Trump Goes Down, He’s Taking Everyone With Him</a></li>
+                    <li><a href="#">What Does Having a Boyfriend Have to Do With Sleep?</a></li>
+                    <li><a href="#">As Impeachment Fights Begin, Administration and Congress Clash Over Deposition</a></li>
+                    <li><a href="#">Finding Balance in a Tiny, Wobbly Boat</a></li>
+                </ul>
+            </div>
+        </div>
+        <footer class="footer-navigation">
+            <div class="top-header">
+                <!-- <div class="border-divider"></div> -->
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.